ios-showcase-template.xcodeproj/project.pbxproj

```diff
@@ -990,17 +964,17 @@
 			baseConfigurationReference = 5185CA4E465C33A6AFF38538 /* Pods-ios-showcase-template.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
 				CODE_SIGN_STYLE = Manual;
-				DEVELOPMENT_TEAM = GHPBX39444;
+				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = "ios-showcase-template/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "org.aerogear.ios-showcase-template";
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE = "cff102ff-c079-4ed6-a8cb-27f8ec14cbd8";
-				PROVISIONING_PROFILE_SPECIFIER = "match InHouse org.aerogear.ios-showcase-template";
+				PROVISIONING_PROFILE = "";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
can we revert some of the changes around the code signing in this file? The settings are required to get it build on circleci.

Reply:
Oh good catch! Will do